doc/classes/TileSet.xml

```diff
@@ -591,7 +591,7 @@
 			Tile coordinates layout where both axis stay consistent with their respective local horizontal and vertical axis.
 		</constant>
 		<constant name="TILE_LAYOUT_STACKED_OFFSET" value="1" enum="TileLayout">
-			Same as [code]TILE_LAYOUT_STAKED[/code], but the first half-offset is negative instead of positive.
+			Same as [code]TILE_LAYOUT_STACKED[/code], but the first half-offset is negative instead of positive.
```

Comment:
Now that https://github.com/godotengine/godot/pull/60275 was merged, this change should be good to apply:

```suggestion
			Same as [constant TILE_LAYOUT_STACKED], but the first half-offset is negative instead of positive.
```